images/router/haproxy/conf/haproxy-config.template

```diff
@@ -46,6 +46,9 @@
 {{- $pathRewriteTargetPattern := `^/.*$` -}}
 
 global
+{{- with (env "ROUTER_HARD_STOP_AFTER") }}
+  hard-stop-after {{env "ROUTER_HARD_STOP_AFTER"}}
```

Comment:
Does the same range apply as for other time values?  Let's validate the value using something like `{{- with $value := clipHAProxyTimeoutValue (firstMatch $timeSpecPattern (env "ROUTER_HARD_STOP_AFTER")) }}`.